app/controllers/hiring_staff/sign_in/base_sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class HiringStaff::SignIn::BaseSessionsController < HiringStaff::BaseController
+  skip_before_action :check_user_last_activity_at
+
+  def end_session_and_redirect
+    notice_key = session[:signing_out_for_inactivity] ? 'signed_out_for_inactivity' : 'signed_out'
+    session.destroy
+    redirect_to new_identifications_path, { notice: I18n.t("messages.access.#{notice_key}") }
```

Comment:
I can't remember if I've asked this before, but I feel like an active decision to sign out should be met with a success notification

Reply:
You're right.